test/Microsoft.AspNet.Mvc.TagHelpers.Test/ValidationMessageTagHelperTest.cs

```diff
@@ -89,19 +111,23 @@ public class ValidationMessageTagHelperTest
             validationMessageTagHelper.ViewContext = expectedViewContext;
 
             // Act & Assert
-            await validationMessageTagHelper.ProcessAsync(context: null, output: output);
+            await validationMessageTagHelper.ProcessAsync(context, output: output);
 
             generator.Verify();
             Assert.Equal("span", output.TagName);
             Assert.Empty(output.Attributes);
-            Assert.Equal("Content of validation message", output.Content);
+            Assert.Equal(expectedPreContent, output.PreContent);
+            Assert.Equal(expectedContent, output.Content);
+            Assert.Equal(expectedPostContent, output.PostContent);
         }
 
         [Theory]
-        [InlineData("Content of validation message", "Content of validation message")]
-        [InlineData("\r\n  \r\n", "New HTML")]
+        [InlineData("Content of validation message", null, null)]
+        [InlineData("Content of validation message", "Some Content", "Some Content")]
+        [InlineData("\r\n  \r\n", "", "New HTML")]
+        [InlineData("\r\n  \r\n", "Some Content", "Some Content")]
```

Comment:
Found issue, I changed the `ValidationMessageTagHelper` to not override whitespace, will undo that change.